lms/authentication/__init__.py

```diff
@@ -0,0 +1,16 @@
+from pyramid.authentication import AuthTktAuthenticationPolicy
+
+from lms.security import groupfinder
+
+
+__all__ = []
+
+
+def includeme(config):
+    config.set_authentication_policy(
+        AuthTktAuthenticationPolicy(
+            config.registry.settings["lms_secret"],
+            callback=groupfinder,
+            hashalg="sha512",
+        )
+    )
```

Comment:
This auth policy is currently used only for logging in to the `/reports` page, in case you were wondering